Download of the Kortex API failed (small enhancement)

Hello !

I tried compiling (with both catkin_make and catkin build) the ros_kortex in a workspace but i had the following error : 
```
CMake Error at top-secret-project/ros_kortex/kortex_driver/CMakeLists.txt:14 (message):
  Download of the Kortex API failed
```

After a quick search, it was because of this line in kortex_driver/CmakeLists.txt : 

```
execute_process(COMMAND ./download_kortex_api.bash
                WORKING_DIRECTORY ../src/ros_kortex/kortex_api/scripts
                RESULT_VARIABLE DOWNLOAD_API_RESULT)

if(NOT DOWNLOAD_API_RESULT EQUAL 0)
  message(FATAL_ERROR "Download of the Kortex API failed")
endif()

```

The ros_kortex "meta-package" is not at the same level as the top-level Cmakelists.txt of my package, so the path  `src/ros_kortex/kortex_api/scripts` is not valid, for me it's `src/top-secret-project/ros_kortex/kortex_api/scripts/`.

Is it possible to change the Cmakelists to add a search condition for the package, to avoid this error (for me and other people using embedded directory) ? Something like this : 

```
find_path (KORTEX_API_DOWNLOADER_PATH  # Set this variable
			download_kortex_api.bash  # To the Working dir of this file 
			${CMAKE_CURRENT_SOURCE_DIR}/../kortex_api/scripts # Search this file here
			../src/ros_kortex/kortex_api/scripts) # and here

if(KORTEX_API_DOWNLOADER_PATH)
	execute_process(COMMAND ./download_kortex_api.bash
			WORKING_DIRECTORY ${KORTEX_API_DOWNLOADER_PATH}
                        RESULT_VARIABLE DOWNLOAD_API_RESULT)
else()
	message(FATAL_ERROR "Could not find download_kortex_api.bash in current workspace")
endif()

if(NOT DOWNLOAD_API_RESULT EQUAL 0)
  message(FATAL_ERROR "Download of the Kortex API failed")
endif()

```

So find_package will search in the same directory as before (from src to the script) + the path from kortex_api to the script

--- a/kortex_driver/CMakeLists.txt
+++ b/kortex_driver/CMakeLists.txt
@@ -5,10 +5,19 @@ project(kortex_driver)
 add_compile_options(-std=c++11)
 add_definitions(-D_OS_UNIX)
 
-# custom bash script to download the API if it has not been downloaded yet
-execute_process(COMMAND ./download_kortex_api.bash
-                WORKING_DIRECTORY ../src/ros_kortex/kortex_api/scripts
-                RESULT_VARIABLE DOWNLOAD_API_RESULT)
+
+find_path (KORTEX_API_DOWNLOADER_PATH  # Set this variable
+			download_kortex_api.bash  # To the Working dir of this file 
+			${PROJECT_SOURCE_DIRl}/../kortex_api/scripts # Search this file here
+			../src/ros_kortex/kortex_api/scripts) # and here
+
+if(KORTEX_API_DOWNLOADER_PATH)
+	execute_process(COMMAND ./download_kortex_api.bash
+			WORKING_DIRECTORY ${KORTEX_API_DOWNLOADER_PATH}
+                        RESULT_VARIABLE DOWNLOAD_API_RESULT)
+else()
+	message(FATAL_ERROR "Could not find download_kortex_api.bash in current workspace")
+endif()
 
 if(NOT DOWNLOAD_API_RESULT EQUAL 0)
   message(FATAL_ERROR "Download of the Kortex API failed")

--- a/kortex_driver/CMakeLists.txt
+++ b/kortex_driver/CMakeLists.txt
@@ -8,8 +8,7 @@ add_definitions(-D_OS_UNIX)
 
 find_path (KORTEX_API_DOWNLOADER_PATH  # Set this variable
 			download_kortex_api.bash  # To the Working dir of this file 
-			${PROJECT_SOURCE_DIRl}/../kortex_api/scripts # Search this file here
-			../src/ros_kortex/kortex_api/scripts) # and here
+			${PROJECT_SOURCE_DIRl}/../kortex_api/scripts) # Search this file here
 
 if(KORTEX_API_DOWNLOADER_PATH)
 	execute_process(COMMAND ./download_kortex_api.bash

--- a/kortex_driver/CMakeLists.txt
+++ b/kortex_driver/CMakeLists.txt
@@ -8,7 +8,7 @@ add_definitions(-D_OS_UNIX)
 
 find_path (KORTEX_API_DOWNLOADER_PATH  # Set this variable
 			download_kortex_api.bash  # To the Working dir of this file 
-			${PROJECT_SOURCE_DIRl}/../kortex_api/scripts) # Search this file here
+			${PROJECT_SOURCE_DIR}/../kortex_api/scripts) # Search this file here
 
 if(KORTEX_API_DOWNLOADER_PATH)
 	execute_process(COMMAND ./download_kortex_api.bash

